src/Symfony/Component/Mime/Encoder/QpContentEncoder.php

```diff
@@ -23,10 +23,6 @@ public function encodeByteStream($stream, int $maxLineLength = 0): iterable
         }
 
         // we don't use PHP stream filters here as the content should be small enough
-        if (stream_get_meta_data($stream)['seekable'] ?? false) {
-            rewind($stream);
```

Comment:
Rewinding is already done before the call, and encoders shouldn't rewind as that makes them less useful in the generic case.